api/v1/checluster_types.go

```diff
@@ -360,6 +361,12 @@ type CheClusterSpecServer struct {
 	WorkspacePodNodeSelector map[string]string `json:"workspacePodNodeSelector,omitempty"`
 	// The pod tolerations put on the workspace pods to limit where the workspace pods can run.
 	WorkspacePodTolerations []corev1.Toleration `json:"workspacePodTolerations,omitempty"`
+	// The default editor to workspace create with.
+	// +optional
+	WorkspaceDefaultEditor string `json:"workspaceDefaultEditor,omitempty"`
```

Comment:
Is this a plugin ID, URI, or either?

Reply:
As I understand it is the same type that we have in [che-server](https://github.com/eclipse-che/che-server) constants - https://github.com/eclipse-che/che-server/blob/main/wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java#L74

Reply:
Hm, I wonder because when using the `che-editor` URL parameter, it's possible to specify a URI to a devfile instead of the usual `publisher/plugin/version` ID. If it's possible to use an URI in the CheCluster field, then that should be mentioned in the doc.

Reply:
I have fixed it.